slack/slack_utils.py

```diff
@@ -57,6 +57,15 @@ def create_channel(channel_name):
 
     return response['channel']['id']
 
+def set_channel_topic(channel_id, channel_topic):
+    response = slack_client.api_call(
+        "channels.setTopic",
+        channel=channel_id,
+        topic=channel_topic
+    )
+
+    return response
```

Comment:
It would be nice if this function checked the response for errors and raised a SlackError if they appear.  The same logic is used in all the other functions and allows us to use exceptions to deal with errors rather than inspecting the response payload wherever the function is used.

Reply:
For sure, that makes sense. I've added some a similar error check as seen in the other functions.